modules/gridBidAdapter.js

```diff
@@ -357,7 +357,8 @@ function _addBidResponse(serverBid, bidRequest, bidResponses) {
         netRevenue: true,
         ttl: TIME_TO_LIVE,
         meta: {
-          advertiserDomains: serverBid.adomain ? serverBid.adomain : []
+          advertiserDomains: serverBid.adomain ? serverBid.adomain : [],
+          networkName: serverBid.ext && serverBid.ext.bidder && serverBid.ext.bidder.grid && serverBid.ext.bidder.grid.demandSource ? serverBid.ext.bidder.grid.demandSource : ''
```

Comment:
Any reason this isn't also called demandSource? Also, should this default to 'grid' ?

May make sense to not send the property if 'demandSource' doesn't exist like this:
`...(serverBid.ext && serverBid.ext.bidder && serverBid.ext.bidder.grid && serverBid.ext.bidder.grid.demandSource && {demandSource: serverBid.ext.bidder.grid.demandSource),`

Reply:
tagging @patmmccann for visibility
I'm assuming networkName vs demandSource is a CafeMedia requirement? we'd rather keep it similar to the actual bid response param though (i.e. demandSource)

as for the default value, right, it should be set to 'grid'

Reply:
Network name is documented here https://github.com/prebid/prebid.github.io/pull/1911

Demandsource is not

Reply:
@patmmccann I suggest using demandSource instead as it's actually breaking down the network into separate demand sources so we should make sure we don't block the publishers and bidders from using both, what do you think?

Reply:
this works for me, i'll do a documentation change as well in that case